.env.example

```
@@ -0,0 +1,6 @@
+NEXT_PUBLIC_VAPID_PUBLIC_KEY=
+VAPID_PUBLIC_KEY=
+VAPID_PRIVATE_KEY=
+GITHUB_TOKEN=
+UMAMI_ANALYTICS_ID=
+NODE_ENV=development
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)